test/SlowTests/Server/Documents/ETL/Snowflake/SnowflakeEtlTests.cs

```diff
@@ -0,0 +1,356 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using FastTests;
+using Orders;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.ETL.Snowflake;
+using Raven.Client.Util;
+using Raven.Server.SqlMigration;
+using Snowflake.Data.Client;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+using TestSnowflakeConnectionString = Tests.Infrastructure.ConnectionString.SnowflakeConnectionString;
+
+namespace SlowTests.Server.Documents.ETL.Snowflake;
+
+public class SnowflakeEtlTests: RavenTestBase
+{
+    public SnowflakeEtlTests(ITestOutputHelper output) : base(output)
```

Comment:
Please add tests for export / import of a database with Snowflake task

Reply:
Also backup and restore (see: https://issues.hibernatingrhinos.com/issue/RavenDB-22627/Missing-database-record-properties-on-full-backup-restore)

Reply:
done